sharedPrompts/.env.example

```
@@ -0,0 +1,58 @@
+﻿# =========================
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)